workspaces/vscode-cpp/README.md

```diff
@@ -0,0 +1,3 @@
+# workspace-vscode-cpp
+
+This image is based on [`prairielearn/workspace-vscode-base`](../vscode-base) with C/C++ tools from [`build-essential`](https://packages.ubuntu.com/noble/build-essential), incuding `g++` and `gcc`. See [`here`](../vscode-base/README.md) for recommended settings when using this workspace.
```

Comment:
Using "here" as link text is an accessibility issue since someone using a screen reader and iterating through all the links won't be able to identify the purpose of the link. Here's an alternative suggestion:

```suggestion
This image is based on [`prairielearn/workspace-vscode-base`](../vscode-base) with C/C++ tools from [`build-essential`](https://packages.ubuntu.com/noble/build-essential), inlcuding `g++` and `gcc`. See [the `prairielearn/workspace-vscode-base` documentation](../vscode-base/README.md) for recommended settings when using this workspace.
```

Tweak if you wish, but then please make the same change to the Python image docs as well.